package.json

```diff
@@ -155,7 +155,7 @@
     "@honeybadger-io/webpack": "^1.2.0",
     "@rails/webpacker": "5.2.1",
     "@testing-library/user-event": "^12.1.7",
-    "ahoy.js": "^0.3.7",
+    "ahoy.js": "^0.3.8",
```

Comment:
refactor (blocking): This looks like a bad merge or maybe ensure you have merged latest as this does not appear to be related to the work being done in this PR.